spec/unit/application/ssl_spec.rb

```diff
@@ -33,12 +33,12 @@
     Puppet[:vardir] = tmpdir("ssl_testing")
 
     # Host assumes ca cert and crl are present
-    File.open(Puppet[:localcacert], 'w') { |f| f.write(@ca_cert.to_pem) }
-    File.open(Puppet[:hostcrl], 'w') { |f| f.write(@crl.to_pem) }
+    File.write(Puppet[:localcacert], @ca_cert.to_pem)
+    File.write(Puppet[:hostcrl], @crl.to_pem)
```

Comment:
I don't know enough ruby to know the implications of this change - are they equivalent?

Reply:
The only difference is `File.write` opens the file in binary mode `wb`. On *nix there isn't any difference between text and binary modes. On Windows, text mode will translate `\n` to `\r\n` when writing, and the reverse when reading. In binary mode, no translation takes place. So there is a minor change here (using binary instead of text), but it's not meaningful here.